Fix `#error Unsupported architecture` in headers of older SDKs and library
path.
--- configure.orig	2023-10-30 22:42:54
+++ configure	2024-05-26 02:06:09
@@ -3774,7 +3774,7 @@
   return 0;
 }
 _ACEOF
-if ac_fn_c_try_compile "$LINENO"
+if false
 then :
   have_arch_x86_64=yes
 else $as_nop
@@ -3793,7 +3793,7 @@
   return 0;
 }
 _ACEOF
-if ac_fn_c_try_compile "$LINENO"
+if false
 then :
   have_arch_arm64=yes
 else $as_nop
@@ -4075,7 +4075,7 @@
 	sparc-sun-*) DAPI_LD_SHARED="\$(LD) -G -Bsymbolic \
 		-o \$@ \$(DECTALK_TTS_OBJS) \$(LIBS)" ;;
 	*-apple-darwin*) DAPI_LD_SHARED="\$(CC) -dynamic -dynamiclib -dylib \
-		-o \$@ \$(DECTALK_TTS_OBJS) \$(LIBS) -Wl,-install_name,libtts_\$(LANG_CODE).so ${UNIVERSAL_BINARY}" ;;
+		-o \$@ \$(DECTALK_TTS_OBJS) \$(LIBS) -Wl,-rpath,@loader_path/../lib/ -Wl,-install_name,@rpath/libtts_\$(LANG_CODE).so ${UNIVERSAL_BINARY}" ;;
 esac
 
 ML_ALL="../build/\$(OS_VERSION)/\$(LANG_CODE)/\$(ML_OUT)/libtts.a ../build/\$(OS_VERSION)/\$(LANG_CODE)/\$(ML_OUT)/libtts.so"
